### DISCUSSION TITLE
Adjusting volume falloff of higher frequencies.

### DISCUSSION THREAD

**Discussion by janwirth** [samhirtarif/react-audio-visualize#14] [Ideas]
As you can see, high frequencies have much lower volume that lower ones:
<img width="233" alt="image" src="https://github.com/samhirtarif/react-audio-visualize/assets/7485959/f18cec94-71f7-4eb1-bb6b-a8483c25d306">

In pink noise, each [octave](https://en.wikipedia.org/wiki/Octave_(electronics)) interval (halving or doubling in frequency) carries an equal amount of noise energy.

It would be nice to have the visualization adjusted for energy rather than volume.

**Comment by samhirtarif** [comment]
Hey - I'll definitely look into this. I will be converting this one into a discussion though under the Ideas section.

**Reply by samhirtarif** [reply]
Any further reading material on this would also be highly appreciated.

**Reply by janwirth** [reply]
is this useful? https://en.wikipedia.org/wiki/Pink_noise I'm not an expert on the subject matter, I am just aware of the phenomenon

**Reply by janwirth** [reply]
I think essentially you want to visualize energy, not just max amplitude.
